www/source/demo/steps/8.html.slim

```diff
@@ -21,7 +21,7 @@ classes: demo_step
         .columns.large-12
           h1.h4="Step 8: It's Build Time!"
         .columns.large-6
-          p You've set up automated builds and Docker container publishing. Now, let's see Habitat Builder in action!
+          p You have set up automated builds and Docker container publishing. Now, let's see Habitat Builder in action!
```

Comment:
`Now, let us`...? :wink:

Reply:
I almost updated it to say that.  Or just deleted `Now, let's see Habitat Builder in action!` 😄